drivers/spi/spi_pl022.c

```diff
@@ -482,8 +482,9 @@ static void spi_pl022_xfer(const struct device *dev)
 		SSP_READ_REG(SSP_DR(cfg->reg));
 
 	while (data->rx_count < chunk_len || data->tx_count < chunk_len) {
+		size_t tx_cnt = 0;
```

Comment:
Please declare in the head of this function.
(The rpi_pico-related code generally follows this convention.)

Reply:
moved the declaration into the head.